tasks/main.yml

```diff
@@ -204,7 +204,7 @@
     dest: "{{ postfix_smtp_sasl_password_map }}"
```

Comment:
Caused an error when `postfix_smtp_sasl_password_map` was left on the default value (empty string).